client/jetbrains/src/test/kotlin/com/sourcegraph/cody/chat/CodyChatMessageHistoryTest.kt

```diff
@@ -1,115 +0,0 @@
-package com.sourcegraph.cody.chat
-import com.intellij.ui.components.JBTextArea
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.BeforeEach
-import org.junit.jupiter.api.Test
-class CodyChatMessageHistoryTest {
```

Comment:
This code is not executed at all and is not part of our build source sets. It's an orphan that was accidentally added while transferring the project to a separate repository or something similar. Even if we add this to source sets, it will not compile due to removed dependencies.